tools/python_api/test/test_arrow.py

```diff
@@ -470,7 +470,7 @@ def test_to_arrow_complex(conn_db_readonly: ConnDB) -> None:
     def _test_node(_conn: kuzu.Connection) -> None:
         query = "MATCH (p:person) RETURN p ORDER BY p.ID"
         query_result = _conn.execute(query)
-        arrow_tbl = query_result.get_as_arrow(12)
+        arrow_tbl = query_result.get_as_arrow()
```

Comment:
I think we should not remove the arg from all of `get_as_arrow` call. I would probably use number, -1, 0, no arg in different test cases to cover all of the branches.

Reply:
What do you think about Alex's earlier comment:
> FYI: I believe they were specified this way in order to validate result acquisition across chunk boundaries.

Do we need to validate the result acquisition across chunk boundaries by specifying a small value for certain cases?

Reply:
I am not sure what this means?

Reply:
Ok, done.